modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -56,6 +56,18 @@ function acquia_cms_site_studio_install() {
         ->save();
     }
   }
+  // Starter "really" needs Google Maps API Keys to "work."
+  // This code is a work around for the dev team to ensure that we can install
+  // these modules w/o issue until a more permanent API key is supplied by
+  // Acquia AND/OR we provide a free maps option.
+  // @todo remove this once the above conditions are met.
+  $gmaps_key = getenv('GMAPS_KEY');
+  if (isset($gmaps_key)) {
```

Comment:
@panshulK This should be `!empty` instead of `isset` and this can go inside function:
`function set_site_studio_credentials` Also, comment should simply something like: Set the google map, if it's present in environment variable. (As starter module is something we use internally and I don't think we've to add comment in our codebase about it).

Reply:
@panshulK Also we've to commit articats file as well for it.